### ISSUE TITLE
Issue with UTF16 decompress 

### ISSUE THREAD

**Issue by pwalimbe** [eduardtomasek/lz-string-python#18]
There seems to be some issue in decompressFromUTF16.. the following code does not work. Is there something wrong?

chk_str = "Test String"
compressed = x.compressToUTF16(chk_str)
print(compressed)
print(x.decompressFromUTF16(compressed))

**Comment by jntsouza** [comment]
Yes, its not working for me too. I'm using base64 instead, but would be better UTF16.
